### PR TITLE
Composer installation explanation.

### PR DESCRIPTION
**Pull request by texpert**
Outlined the Docker and Composer installation paths and added the commands for db initialization and server run into Composer section.

### CODE REVIEW

File: distribution/index.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ asynchronous jobs to your APIs is very straightforward.
 
 ## Installing the framework
 
+##### In Docker container
```
Comment (dunglas) [inline distribution/index.md]: Can you make it a level 3 title please? `###`

File: distribution/index.md [hunk 2]
```diff
@@ -83,9 +85,24 @@ However, Doctrine ORM is definitely the easiest way to persist and query data in
 included in the Standard Edition. This Doctrine ORM bridge is optimized for performance and development convenience. Doctrine
 ORM and its bridge supports major RDBMS including MySQL, PostgreSQL, SQLite, SQL Server and MariaDB.
 
-Note: Instead of using Docker, API Platform can also be installed on the local machine using [Composer](https://getcomposer.org/):
+##### Via Composer
```
Comment (dunglas) [inline distribution/index.md]: Same here `###`

File: distribution/index.md [hunk 3]
```diff
@@ -83,9 +85,24 @@ However, Doctrine ORM is definitely the easiest way to persist and query data in
...
+    
+    $ cd bookshop-api
+    $ php bin/console doctrine:database:create
+    $ php bin/console doctrine:schema:create
+    
+And start the server:    
+    
+    $ php bin/console server:run
+
+
```
Comment (dunglas) [inline distribution/index.md]: Can you remove the 2 extra lines?

**Comment by dunglas** [comment]
Great improvement thanks! I just added a few comments related to markdown.

**Comment by texpert** [comment]
Thank you for awesome platform!

**Pull request merged**